modules/system-view/src/main/java/org/apache/ignite/internal/systemview/utils/SystemViewUtils.java

```diff
@@ -49,7 +53,7 @@ public static CreateSystemViewCommand toSystemViewCreateCommand(SystemView<?> vi
                     ColumnParams.builder()
                             .name(((NodeSystemView<?>) view).nodeNameColumnAlias())
                             .type(ColumnType.STRING)
-                            .length(CatalogUtils.DEFAULT_VARLEN_LENGTH)
+                            .length(NODE_NAME_FIELD_LENGTH)
```

Comment:
Maybe we should set nullability of node alias column to `false`?

Reply:
good catch! fixed